Cogs/StudentCommands.py

```diff
@@ -144,3 +144,20 @@ async def ping(self, ctx):
         latency = round(self.bot.latency * 1000)
         await ctx.send(f'{latency} ms')
         await log(self.bot, f'{ctx.author} pinged from #{ctx.channel}, response took {latency} ms')
+
+    @commands.command()
+    async def attendance(self, ctx):
+
+        # The user object does not have a .voice.channel attribute so simply using author.id.voice does not work here
+        try:
+            channel = ctx.guild.get_member(ctx.message.author.id).voice.channel
```

Comment:
If you ever need to get a specific member object, please_ use get_member() from utils.py. The builtin get_member() function is a direct API call and can take a long time to run.

Also note that the context object provides
`discord.ext.commands.Context.author.voice.channel`
This should negate the need for a get_member() call entirely